release/OpenWrt/Makefile

```diff
@@ -0,0 +1,89 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=hetzner_ddns
+PKG_VERSION:=0.2.5
+PKG_RELEASE:=1
+PKG_LICENSE:=BSD-2-Clause-license
+PKG_MAINTAINER:=Filip Parag
+
+# clone sources
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/filiparag/hetzner_ddns.git
+PKG_SOURCE_VERSION:=master
```

Comment:
I think this should be set to `0.2.5` tag, as `PKG_VERSION` is not following development branch `master`. Correct me if I'm wrong.

Reply:
Yes, of course you are right. This is fixed now with [bea11d1](https://github.com/filiparag/hetzner_ddns/pull/18/commits/bea11d13da88fc315202d75d320456a2b4a1dcd4).